tests/stub/authorization.py

```diff
@@ -238,7 +238,7 @@ def test_should_fail_with_auth_expired_on_pull_using_session_run(
 
         session = driver.session('r', database=self.get_db())
         try:
-            session.run("RETURN 1 as n")
+            session.run("RETURN 1 as n").consume()
```

Comment:
I'd suggest to split the two calls into two statements:
```
res = session.run("RETURN 1 as n")
res.consume()
```

This might make it easier to debug which call caused an error, should one be raised.